docs/UserGuide.md

```diff
@@ -40,95 +50,163 @@ Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+#### Adding a task: `add`
+Adds a task to GGist<br>
+General format: `add TASK, [DATE], [TIME], [PRIORITY], [FREQUENCY]`
 
-> Persons can have any number of tags (including 0)
+<img src="images/2ndimpt.png" width="500"><br>
```

Comment:
try including this information in the document text rather than an image. with text, you can edit it as and when you want to make changes.